src/core/default/commands/relationaldb/shell.py

```diff
@@ -23,15 +22,67 @@ def add_arguments(self, parser: ArgumentParser) -> None:
             type=str,
             help="The database to execute on. Name must include component name. ex: comp1.myDb",
         )
+        parser.add_argument("--c", nargs="+", type=str, help="sql command to execute")
+        parser.add_argument("--command", nargs="+", type=str, help="sql command to execute")
```

Comment:
You can add the `--command` flag to the above line. That way arg parse know it is the same option just a different way of writing it https://docs.python.org/3/library/argparse.html#the-add-argument-method

This way the help message looks like 
```
optional arguments:
  -h, --help            show this help message and exit
  --c C [C ...], --command C [C ...]
                        sql command to execute
  --f F [F ...]         execute sql commands from a file
```
as opposed to 
```
optional arguments:
  -h, --help            show this help message and exit
  --c C [C ...]         sql command to execute
  --command COMMAND [COMMAND ...]
                        sql command to execute
  --f F [F ...]         execute sql commands from a file
```